lib/Runtime/Base/ScriptContext.cpp

```diff
@@ -375,7 +375,7 @@ namespace Js
 
     ScriptContext::~ScriptContext()
     {
-        Assert(isFinalized || !isInitialized);
+        Assert(isFinalized);
```

Comment:
then we need another state to know that we are half way in JsrtContext initialization code path...